src/blobs/BlobRefRegistry.sol

```diff
@@ -12,6 +12,7 @@ contract BlobRefRegistry is IBlobRefRegistry {
     function registerRef(uint256[] calldata blobIndices) external returns (bytes32 refHash, BlobRef memory ref) {
         ref = _getRef(blobIndices);
         refHash = _registerRefHash(ref);
+        emit Registered(refHash, ref);
```

Comment:
I removed this event in PR 38 because it is emitted in `_registerRefHash`. Are you intentionally adding it back (because I misunderstood something) or was this just an artifact of the parallel PRs?

Reply:
I added it back intentionally as I feel it will be more convenient for apps to subscribe to event from just this contract to get the ref objects.

Reply:
Since this is the only outstanding issue, I'll merge the PR first then we can change it later if you prefer this event should be removed.